### DISCUSSION TITLE
Add an option to switch color themes when switching between workspaces.

### DISCUSSION THREAD

**Discussion by YousufSSyed** [Floorp-Projects/Floorp#434] [Feature Request]
I have different workspaces for home and school, and would like to feel like I'm using a different browser when I switch workspaces, a psychological separation between them. The [Firefox Color](https://addons.mozilla.org/en-US/firefox/addon/firefox-color/) addon can be used to change color themes. Could Floorp have a setting to switch them when workspaces are switched?

**Comment by llba5313** [comment]
I would like to see this also - it would help differentiate which workspace we are in at the moment!
